gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -118,7 +118,7 @@ def run(
         # transforming the result to the list of Point3
         wti_list = [None] * num_images
         for i in range(num_images):
-            if wRi_list[i] is not None:
+            if wRi_list[i] is not None and wti_values.exists(i):
```

Comment:
`wRi_list[i] is not None` is redundant?

Reply:
true, but i think it's a bit more readable?

Reply:
I think it makes it more confusing. But i am fine with it.